ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -160,30 +164,24 @@ else
 	#
 	# Set AUTOCNF flag to use it in AUTOSTART scripts.
 	#
-	if param compare SYS_AUTOCONFIG 1
+	if param greater SYS_AUTOCONFIG 0
 	then
-		# Wipe out params except RC*, flight modes, total flight time, accel cal, gyro cal
-		param reset_nostart RC* COM_FLTMODE* LND_FLIGHT_T_* TC_* CAL_ACC* CAL_GYRO*
-		set AUTOCNF yes
-	else
-		if param compare SYS_AUTOCONFIG 2
-		then
-			set AUTOCNF yes
-		else
-			set AUTOCNF no
+		if param compare SYS_AUTOCONFIG 1
+			# Wipe out params except RC*, flight modes, total flight time, accel cal, gyro cal
+			param reset_nostart RC* COM_FLTMODE* LND_FLIGHT_T_* TC_* CAL_ACC* CAL_GYRO*
 		fi
+
+		set AUTOCNF yes
 	fi
 
 	#
 	# Optional board defaults: rc.board_defaults
 	#
-	set BOARD_RC_DEFAULTS /etc/init.d/rc.board_defaults
```

Comment:
Keeping the scope of these more local is IMO preferrable, as it makes it easier to track the variable state, and reduces memory usage.

Reply:
Hi@bkueng! Thanks for your feedback. I understand your point and agree with keeping variables as close as possible to where they are used as a general rule to follow.  The trouble with the startup scripts is accounting for variables to ensure they get `unset`.   For variables that get set/unset within a few LOC, this is easy, but for variables that are used by multiple startup files this becomes a headache.  I realize that for this PR the set/unsets are close together, I am primarily motivated by employing a standard practice that is easily identifiable and helps reduce errors in the future in this PR.

Reply:
(It's totally acceptable if you'd prefer to keep the existing.  Not a problem at all to close out this PR unmerged, I understand where you are coming from.)

Reply:
I generally agree, but for these specific instances I prefer the existing way (for the same reasons you already mention).